napari/_tests/test_status.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+
+from napari.utils.status_messages import status_format
+
+
+def test_status_format(make_napari_viewer):
```

Comment:
```suggestion
def test_status_format():
```
This test doesn't need the napari viewer